### ISSUE TITLE
"all" config

### ISSUE THREAD

**Issue by brettz9** [mysticatea/eslint-plugin#21]
Hi,

I'm using your useful plugin from my config [eslint-config-ash-nazg](https://github.com/brettz9/eslint-config-ash-nazg) and was wondering if you might add a config which includes all of the rules of this plugin (but no other rules from eslint or other plugins/configs).

This would assist me now and into the future for auto-comparing my config to the configs I am extending.

Thanks!

**Comment by mysticatea** [comment]
Thank you for your proposal.

If the purpose was comparison, I wonder if `Object.keys(require("@mysticatea/eslint-plugin").rules)`  is enough?

(as a side note, for a reference, there is an RFC for `ConfigTester` that can check a config contains all rules or not: https://github.com/eslint/rfcs/pull/27)

**Comment by brettz9** [comment]
Yeah sorry, I amended my original post to clarify my intent. While it might streamline my comparing (since I'm comparing with configs in most cases), I'd really just like to make my source clearer and have one extends for either an "all" or "recommended" rule. I figure those already familiar with other configs such as yours won't need to be troubled to see which individual ones I'm using if I am merely extending, and then I'd also get the benefit of any other rules you added (though I review updates to affirm I agree with the additions, I tend to find that smart people's "recommended" rules are worth my being forced to examine for inclusion (unlike if I only whitelisted individual rules), and it'd be nice not to have to add rules individually with every update except to override as I felt necessary).

Great to know about the rfc, I hope to take a closer look as I have a chance.

**Comment by MichaelDeBoey** [comment]
Hi @brettz9!

Since this repo is unmaintained, you might want to re-open this issue in the @eslint-community fork https://github.com/eslint-community/eslint-plugin-mysticatea

For more info about why we created this organization, you can read https://eslint.org/blog/2023/03/announcing-eslint-community-org
